app/page/send/index.tsx

```diff
@@ -31,97 +86,130 @@ const SelectToken = ({ setTokenSelected }) => {
   )
 }
 
-const SelectAddress = ({ to, setTokenSelected, onChangeTo }) => {
-  const [invalidTo, setInvalidTo] = useState<boolean>(false)
-  const account = useAccount()
+const SelectAddress = ({ onStepBack, setTxTo }) => {
+  const [inputTo, setInputTo] = useState<HexString>("")
+  const accounts = useAccounts()
 
   const handleToChange = async (event: ChangeEvent<HTMLInputElement>) => {
     const value = event.target.value
-    onChangeTo(value)
-    try {
-      console.log(`${ethers.getAddress(value)}`)
-      setInvalidTo(false)
-    } catch (error) {
-      console.log(`Invalid to`)
-      setInvalidTo(true)
-    }
+    setInputTo(value)
   }
   return (
     <>
-      <StepBackHeader
-        title="Select Address"
-        onStepBack={() => {
-          setTokenSelected(null)
-          navigate(Path.Send)
-        }}
-      />
-      <div className="flex">
-        <label className="flex-1">To:</label>
+      <StepBackHeader title="Select Address" onStepBack={onStepBack}>
         <input
           type="text"
           id="to"
-          value={`${to}`}
+          value={inputTo}
           onChange={handleToChange}
-          list="suggestionTo"
-          className={`border w-96 outline-none ${
-            invalidTo ? "border-red-500" : "border-gray-300"
-          }`}></input>
-        <datalist id="suggestionTo">
-          <option value={account.address}></option>
-        </datalist>
+          className="width-full border-solid border-black border-[2px] rounded-[16px] p-[16px] text-[16px]"
+          placeholder="Enter address"
+          required
+        />
+      </StepBackHeader>
+      <div className="py-[24px] h-[311px]">
```

Comment:
I accidentally found this list should be overflow scrollable because my wallet has more than 4 accounts 🙈

Reply:
[replace w-full and pr with min-w-0](https://github.com/pilagod/waallet-extension/pull/172/commits/951101e5fde56053d1fa344b46d9095e6c51d9a2)

https://github.com/user-attachments/assets/5ad69687-4b35-4993-865a-d9d62ec6c843